charts/ethos-repl-docmosis-backend/values.preview.template.yaml

```diff
@@ -7,11 +7,6 @@ java:
     ETHOS_REPL_DB_NAME: "{{ .Values.postgresql.postgresqlDatabase}}"
     ETHOS_REPL_DB_USER_NAME: "{{ .Values.postgresql.postgresqlUsername}}"
     ETHOS_REPL_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
-  keyVaults:
```

Comment:
You would still need this :
1. To override defaults in values.yaml , as keyvaults will take precedence over env vars.

Reply:
Added that back